src/Type/PostObject/Mutation/PostObjectMutation.php

```diff
@@ -0,0 +1,318 @@
+				],
+				'date'          => [
+					'type'        => Types::string(),
+					'description' => esc_html__( 'The date of the object', 'wp-graphql' ),
+				],
+				'dateGmt'       => [
+					'type'        => Types::string(),
+					'description' => esc_html__( 'The date (in GMT zone) of the object', 'wp-graphql' ),
+				],
+				'excerpt'       => [
+					'type'        => Types::string(),
+					'description' => esc_html__( 'The excerpt of the object', 'wp-graphql' ),
+				],
+				'menuOrder'     => [
+					'type'        => Types::int(),
+					'description' => esc_html__( 'A field used for ordering posts. This is typically used with nav menu items or for special ordering of hierarchical content types.', 'wp-graphql' ),
+				],
+				'mimeType'      => [
+					'type'        => Types::mime_type_enum(),
+					'description' => esc_html__( 'If the post is an attachment or a media file, this field will carry the corresponding MIME type. This field is equivalent to the value of WP_Post->post_mime_type and the post_mime_type column in the `post_objects` database table.', 'wp-graphql' ),
+				],
+				'modified'      => [
+					'type'        => Types::string(),
+					'description' => esc_html__( 'The local modified time for a post. If a post was recently updated the modified field will change to match the corresponding time.', 'wp-graphql' ),
+				],
+				'modifiedGmt'   => [
+					'type'        => Types::string(),
+					'description' => esc_html__( 'The GMT modified time for a post. If a post was recently updated the modified field will change to match the corresponding time in GMT.', 'wp-graphql' ),
+				],
+				'parentId'      => [
+					'type'        => Types::id(),
+					'description' => esc_html__( 'The ID of the parent object', 'wp-graphql' ),
+				],
+				'password'      => [
+					'type'        => Types::string(),
+					'description' => esc_html__( 'The password used to protect the content of the object', 'wp-graphql' ),
+				],
+				'pinged'        => [
+					'type'        => Types::list_of( Types::string() ),
+					'description' => esc_html__( 'URLs that have been pinged.', 'wp-graphql' ),
+				],
+				'pingStatus'    => [
+					'type'        => Types::string(),
+					'description' => esc_html__( 'The ping status for the object', 'wp-graphql' ),
+				],
+				'slug'          => [
+					'type'        => Types::string(),
+					'description' => esc_html__( 'The slug of the object', 'wp-graphql' ),
+				],
+				'status'        => [
+					'type'        => Types::post_status_enum(),
+					'description' => esc_html__( 'The status of the object', 'wp-graphql' ),
+				],
+				'title'         => [
+					'type'        => Types::string(),
+					'description' => esc_html__( 'The title of the post', 'wp-graphql' ),
+				],
+				'toPing'        => [
+					'type'        => Types::list_of( Types::string() ),
+					'description' => esc_html__( 'URLs queued to be pinged.', 'wp-graphql' ),
+				],
+			];
```

Comment:
ya, I'll need to play with how input/output fields can be re-used better since there is definitely overlap in the type/descriptions. 

I'll create a story to look into how that might look. 

There won't always be a 1-to-1 match between what input fields there are and what output fields there are though, so we'll just have to keep that into consideration whatever we come up with. . .but I agree, we could clean up some repetitive code in some way for sure.